Makefile.in

```diff
@@ -379,6 +379,7 @@ PY_PROGRAMS = \
 	nodist/fuzzer/actual_fail \
 	nodist/fuzzer/generate_dictionary \
 	nodist/install_test \
+	nodist/clang_check_attributes.py \
```

Comment:
Move up to keep sorted please.  Also I would suggest drop .py extension, as it's an executable and the language doesn't really matter.